var/spack/repos/builtin/packages/libjwt/package.py

```diff
@@ -21,4 +21,6 @@ class Libjwt(AutotoolsPackage):
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
+    # Needs openssl at runtime to ensure we can generate keys
+    depends_on('openssl')
```

Comment:
Is this only a `run` dependency or , maybe, `('build', 'run')`?

Reply:
Yes. build, run.

Reply:
Never hurts to be explicit.  Thanks.